### ISSUE TITLE
Halyard fails to deploy gate with http readiness probe

### ISSUE THREAD

**Issue by fraenkel** [spinnaker/spinnaker#4568]
### Issue Summary:
Deploying spinnaker via the helm chart fails once ingress is enabled

### Cloud Provider(s):
Kubernetes

### Description:
deploy spin-gate fails when ingress is enabled

### Additional Details:

The relevant output:
```
- Deploy spin-rosco
  Failure
Problems in Global:
! ERROR Unexpected exception: while parsing a flow mapping
in 'string', line 50, column 14:
"httpGet": {
^
expected ',' or '}', but got <scalar>
in 'string', line 53, column 5:
"scheme": "HTTP"
^
```

The bug is in 
https://github.com/spinnaker/halyard/blob/master/halyard-deploy/src/main/resources/kubernetes/manifests/httpReadinessProbe.yml#L4

There is a missing comma.

**Comment by romil-punetha** [comment]
What docker image version contains this fix? Im using 1.21.0 and 1.21.1 and both have this bug

**Comment by romil-punetha** [comment]
This was fixed by using `gcr.io/spinnaker-marketplace/halyard:nightly`.

**Comment by ezimanyi** [comment]
This was cherry-picked into 1.21.x but had not yet made it into an actual release.  I've just released 1.21.2 which contains the fix.
